cds_ils/importer/config.py

```diff
@@ -0,0 +1,13 @@
+CDS_ILS_IMPORTER_RECORD_TAG = "//*[local-name() = 'record']"
+
+CDS_ILS_IMPORTER_PROVIDERS = {
+    'cds': {'priority': 1,
+            'agency_code': 'SZeGeCERN',
```

Comment:
is `agency_code` something defined in MarcXML?

Reply:
yes it is an id of the agency keeping the repository, every XML coming from the providers has its own code